osis-core/src/main/java/com/scality/osis/service/ScalityOsisService.java

```diff
@@ -6,10 +6,67 @@
 
 package com.scality.osis.service;
 
-import com.vmware.osis.model.OsisS3Credential;
-import com.vmware.osis.service.OsisService;
+import com.scality.osis.model.*;
+import com.scality.osis.model.ScalityOsisCaps;
+import com.scality.osis.model.OsisS3Credential;
```

Comment:
Are the two last lines needed if you import all the namespace with the star?

Reply:
you are right, it was very intentional to not change that.
We will refactor the whole code base again soon. For now, so close to the release, I didn't want to add more complexities in the review.